r2r/main/api/client.py

```diff
@@ -180,48 +162,37 @@ def ingest_files(
             for _, file_tuple in files_to_upload:
                 file_tuple[1].close()
 
-    @monitor_request
-    def update_documents(
-        self,
-        documents: list[dict],
-        versions: Optional[list[str]] = None,
-        metadatas: Optional[list[dict]] = None,
-    ) -> dict:
-        request = R2RUpdateDocumentsRequest(
-            documents=documents, versions=versions, metadatas=metadatas
-        )
-        return self._make_request(
-            "POST", "update_documents", json=json.loads(request.json())
-        )
-
     @monitor_request
     def update_files(
         self,
-        files: list[str],
+        file_paths: list[str],
```

Comment:
I'll update the js client to reflect changes here.

Reply:
great, ty This one should just be cosmetic.

Reply:
also note that user ids is no longer a passed field